src/droplet.F

```diff
@@ -354,7 +354,7 @@ subroutine droplet_driver(dt,dbldt,mtime,xh,uh,ruh,xf,yh,vh,rvh,yf,zh,mh,rmh,zf,
 #ifdef _B4B01F
     !$acc update &
     !$acc host(pdata,xf,yf,zf,zh,sigma,sigmaf, &
-    !$acc      ua,va,wa,qa,ta,rho,prs,znt,qten,thten,zs)
```

Comment:
@sjsprecious Please note that George appears to have forgotten to add `uten`, `vten`, `wten` here when he updated. This could be a reason why the GPU and CPU versions didn't match? I replaced these with `dpten`, which is hopefully correct.

Reply:
Gotcha, didn't notice that

Reply:
@johnmauff you both obviously have more expertise than I do, but looking back through, I never see `dpten` being updated on either host or device. How can this be the case? Am I missing it somewhere, or perhaps this is somehow automatically accounted for with the `acc atomic update` above each of the `dpten` updates in `project_feedback`? If not, I don't see how these GPU atomic calculations for `dpten` ever make their way back to the CPU before being used inside solve3.F